UI/ChatUI/ChatUI/ChatUI/MainPage.xaml

```diff
@@ -0,0 +1,427 @@
+﻿<Page
+	x:Class="ChatUI.MainPage"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:android="http://uno.ui/android"
+	xmlns:behaviors="using:ChatUI.Behaviors"
+	xmlns:converters="using:ChatUI.Converters"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:ios="http://uno.ui/ios"
+	xmlns:local="using:ChatUI"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:xamarin="http://uno.ui/xamarin"
+	Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+	mc:Ignorable="d ios android xamarin">
+
+	<Page.Resources>
+		<ResourceDictionary>
+			<converters:FromNullableBoolToVisibilityConverter x:Key="BooleanToVisibility" VisibilityIfTrue="Visible" />
+
+			<Style TargetType="ListView">
+				<Setter Property="RenderTransformOrigin" Value=".5,.5" />
+				<Setter Property="RenderTransform">
+					<Setter.Value>
+						<ScaleTransform ScaleY="-1" />
+					</Setter.Value>
+				</Setter>
+			</Style>
+			<Style TargetType="ListViewItem">
+				<Setter Property="Template">
+					<Setter.Value>
+						<ControlTemplate TargetType="ListViewItem">
+							<Grid
+								x:Name="ContentBorder"
+								Background="{TemplateBinding Background}"
+								BorderBrush="{TemplateBinding BorderBrush}"
+								BorderThickness="{TemplateBinding BorderThickness}"
+								CornerRadius="{TemplateBinding CornerRadius}"
+								RenderTransformOrigin=".5,.5">
+								<Grid.RenderTransform>
+									<!--  Setting the RenderTransform on the ListViewItem itself is not supported by Uno, we need to set it on the first child instead  -->
+									<ScaleTransform ScaleY="-1" />
+								</Grid.RenderTransform>
+								<Grid
+									x:Name="ContentPresenterGrid"
+									Margin="0,0,0,0"
+									Background="Transparent">
+									<Grid.RenderTransform>
+										<TranslateTransform x:Name="ContentPresenterTranslateTransform" />
+									</Grid.RenderTransform>
+									<ContentPresenter
+										x:Name="ContentPresenter"
+										Margin="{TemplateBinding Padding}"
+										HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
+										VerticalAlignment="{TemplateBinding VerticalContentAlignment}"
+										xamarin:ContentTemplateSelector="{TemplateBinding ContentTemplateSelector}"
+										Content="{TemplateBinding Content}"
+										ContentTemplate="{TemplateBinding ContentTemplate}"
+										ContentTransitions="{TemplateBinding ContentTransitions}" />
+								</Grid>
+								<!--
+									The 'Xg' text simulates the amount of space one line of text will occupy.
+									In the DataPlaceholder state, the Content is not loaded yet so we
```

Comment:
`In the DataPlaceholder state`..? that seems like a leftover from other project
please update the comment or remove it

Reply:
Yep, this was leftover from code borrowed from Steve. Removed.